src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -498,16 +505,27 @@ export class MgtPersonCard extends MgtTemplatedComponent {
   }
 
   /**
-   * Use the sip: protocol to initiate a chat message to the user.
+   * Initiate a chat message to the user via deeplink.
    *
    * @protected
    * @memberof MgtPersonCard
    */
   protected chatUser() {
     const user = this.personDetails as MicrosoftGraph.User;
-    if (user && user.mailNickname) {
-      const chat = user.mailNickname;
-      window.open('sip:' + chat, '_blank');
+    if (user && user.userPrincipalName) {
+      const users: string = user.userPrincipalName;
+      const url = `https://teams.microsoft.com/l/chat/0/0?users=${users}`;
+      const openWindow = () => window.open(url, '_blank');
+
+      if (TeamsProvider.isAvailable) {
```

Comment:
I think this is going to cause an issue if/when we split up the packages and the providers go in a separate package. I think you should factor this into it's own Teams helper that's used by the TeamsProvider and this component?

Reply:
Agh, but that might be a breaking change for the Teams provider. A way to avoid the breaking change could be to keep the static property on TeamsProvider and deprecate it. You will probably have to create setters and getters on the TeamsProvider to set the right property in the helper

Reply:
Check out the latest changes.

I moved `isAvailable`, `executeDeepLink`, and `microsoftTeamsLib` to a new `TeamsHelper` class, and fixed `TeamsProvider` to use the new helper. This way the components can refer to `TeamsHelper` instead of `TeamsProvider` to determine Teams availability and any other Teams lib interactions (such as deeplinking).